Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM registry.opensuse.org/opensuse/leap:latest
```

Comment:
but let's stick to some certain version here ( 15.3 )

Reply:
Ok